localstack/utils/bootstrap.py

```diff
@@ -457,6 +457,28 @@ def _cfg(cfg: ContainerConfiguration):
 
         return _cfg
 
+    @staticmethod
+    def publish_dns_ports(cfg: ContainerConfiguration):
+        dns_ports = [
+            Port(config.DNS_PORT, protocol="udp"),
+            Port(config.DNS_PORT, protocol="tcp"),
+        ]
+        if container_ports_can_be_bound(dns_ports, address=config.DNS_ADDRESS):
+            # expose the DNS server to the host
+            # TODO: update ContainerConfiguration to support multiple PortMappings objects with different bind addresses
+            docker_flags = []
+            for port in dns_ports:
+                docker_flags.extend(
+                    [
+                        "-p",
+                        f"{config.DNS_ADDRESS}:{port.port}:{port.port}/{port.protocol}",
+                    ]
+                )
+            if cfg.additional_flags is None:
+                cfg.additional_flags = " ".join(docker_flags)
+            else:
+                cfg.additional_flags += " ".join(docker_flags)
```

Comment:
When appending, we need to append a space in between as well!

Reply:
Yeah thanks, just saw that in the test failure